crates/forge/tests/cli/soldeer.rs

```diff
@@ -7,17 +7,14 @@ use std::{
 
 use foundry_test_utils::forgesoldeer;
 use std::io::Write;
-
 forgesoldeer!(install_dependency, |prj, cmd| {
     let command = "install";
     let dependency = "forge-std~1.8.1";
 
     let foundry_file = prj.root().join("foundry.toml");
 
-    cmd.arg("soldeer").args([command, dependency]).assert_success().stdout_eq(str![[r#"
-🦌 Running [..]oldeer install 🦌
-...
-"#]]);
+    cmd.arg("soldeer").args([command, dependency]);
+    cmd.execute();
```

Comment:
We would like to use Snapbox where possible

Generic checks on whether the command has ran succesfully can use `.assert_success()` rather than `.execute()`

Reply:
done